src/ru/skypro/Main.java

```diff
@@ -3,6 +3,92 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+//Task 1
+	byte clientOs = 1;
+    if (clientOs == 0) {
+        System.out.println("Установите версию приложения для iOS по ссылке");
+    }
+    if (clientOs == 1) {
+        System.out.println("Установите версию приложения для Android по ссылке");
+    }
+
+//Task 2
+    int clientDeviceYear = 2014;
+    if (clientOs == 0) {
+        if (clientDeviceYear < 2015){
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        }else
+            System.out.println("Установите версию приложения для iOS по ссылке");
+    }else {
+        if (clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        }else
+            System.out.println("Установите версию приложения для Android по ссылке");
+    }
+
+//Task 3
+    int year = 2021;
+    if (((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0)) {
+        System.out.println(year + " год является високосным");
+    }else
+        System.out.println(year + " год не является високосным");
+
+//Task 4
+    int deliveryDistance = 95;
+    int deliveryDays = deliveryDistance / 40 + 1;
+    if (deliveryDays == 1) {
+        System.out.println("Потребуется " + deliveryDays + " день для доставки.");
+    } else{
+        if ((deliveryDays >= 2) && (deliveryDays <= 4))
+            System.out.println("Потребуется " + deliveryDays +" дня для доставки.");
+        else
+            System.out.println("Потребуется " + deliveryDays +" дней для доставки.");
+    }
+
+//Task 5
+    byte monthNumber = 12;
+    switch (monthNumber){
+        case 1:
+            System.out.println("В этом месяце на Руси зима");
+            break;
+        case 2:
+            System.out.println("В этом месяце на Руси зима");
+            break;
+        case 3:
+            System.out.println("В этом месяце на Руси весна");
+            break;
+        case 4:
+            System.out.println("В этом месяце на Руси весна");
+            break;
+        case 5:
+            System.out.println("В этом месяце на Руси весна");
```

Comment:
поскольку для смежных по сезону месяцев мы ожидаем один и тот же результат, то break между ними можно убрать, например:
```suggestion
        case 3:
            System.out.println("В этом месяце на Руси весна");
        case 4:
            System.out.println("В этом месяце на Руси весна");
        case 5:
            System.out.println("В этом месяце на Руси весна");
            break;
```